Bio/PDB/PICIO.py

```diff
@@ -391,12 +391,15 @@ def default_dihedron(ek: List, ric: IC_Residue) -> None:
                 )
 
             if paKey in da:
+                angl = da[paKey] + dihedra_secondary_defaults[rdclass][1]
+                angl = angl if (angl <= 180.0) else angl - 360.0
+                angl = angl if (angl >= -180.0) else angl + 360.0
```

Comment:
If you're using this operation often (I see it below at least once more), I'd make it a function.